src/redux/selectors/walletSelectors.js

```diff
@@ -44,7 +44,7 @@ const selectorTotalBalance = createSelector(
 const selectorTotalLLM = createSelector(
   selectorBalances,
   (reducer) => (
-    reducer.meritsTotalAmount.amount
+    reducer?.meritsTotalAmount?.amount
```

Comment:
Ditto, `reducer` and `reducer.meritsTotalAmount` should never be empty, if they are then we have some serious issue somewhere.